tests/test_ppo_trainer.py

```diff
@@ -230,3 +230,42 @@ def test_ppo_step_with_ref_and_custom_layers_warning(self):
                 dataset=dummy_dataset,
                 num_shared_layers=num_shared_layers,
             )
+
+    def test_ppo_step_rewards_shape(self):
```

Comment:
can we also add a test that checks the query/response shape?

Reply:
Addressed in [e3d4e26](https://github.com/lvwerra/trl/pull/76/commits/e3d4e26e118baf52bebe6a42b0285447ab17505b)